### PR TITLE
Fill in illustration text

### PR DESCRIPTION
**Pull request by apragsdale**
Add some text to the illustration example in the main doc.

### CODE REVIEW

**Review by grahamgower — APPROVED** [review]
LGTM!

File: paper.tex [hunk 1]
```diff
@@ -370,6 +360,35 @@ \section*{Application: simulation using Demes}
     \label{fig:showcase}
 \end{figure}
 
+Here, we highlight the interface between Demes and other software, including
+simulation and model illustration tools. Demes allows us to specify a
+demographic model a single time, which can be used as the input for a growing
```
Comment (jeromekelleher) [inline paper.tex]:
```suggestion
demographic model which can be used as the input for a growing
```

File: paper.tex [hunk 2]
```diff
@@ -370,6 +360,35 @@ \section*{Application: simulation using Demes}
...
+number of simulation software (Table~\ref{tab:software}). We implemented the
```
Comment (jeromekelleher) [inline paper.tex]:
```suggestion
number of simulation packages (Table~\ref{tab:software}). We implemented the
```

File: paper.tex [hunk 3]
```diff
@@ -370,6 +360,35 @@ \section*{Application: simulation using Demes}
...
+at time zero (i.e., present day) to obtain the joint site-frequency spectrum
+(SFS), a summary of observed allele frequencies widely used in evolutionary
+inference
+\citep{bustamante2001directional,gutenkunst2009inferring,tennessen2012evolution,jouganous2017inferring,kamm2017efficient,kim2017inference}.
+We used \msprime\ \citep{baumdicker2021-iu} to simulate a large recombining
+region with the mutation rate assumed in \citet{tennessen2012evolution} and
+computed the observed SFS using \tskit\ \citep{ralph2020efficiently}. We
+compared that simulated SFS to its expectation computed using \moments\
+\citep{jouganous2017inferring} (Figure~\ref{fig:showcase}B,C). For both
+\msprime\ and \moments\ simulations, the same demographic model was used as
```
Comment (jeromekelleher) [inline paper.tex]: Maybe be more concrete here, "the Demes model shown in Fig XX was used as input"...

Just to hammer home the point that it's precisely this text description that's input.

**Review by jeromekelleher — APPROVED** [review]
Very nice. Some minor tweaks suggested.

File: paper.tex (hunk 3 above)
Comment (apragsdale) [reply paper.tex]: I edited this paragraph quite a bit based on your suggestion. Let me know what you think! Thanks for the suggestions

**Comment by molpopgen** [comment]
This looks great to me!

**Comment by jeromekelleher** [comment]
Looks great @apragsdale, merge whenever you're happy!

**Pull request merged**